content/en/about/community/toc/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Technical Oversight Committee
+description: Information describing the technical oversight committee within the Istio Community
+weight: 10
```

Comment:
This weight is indeterminate with `Join`.

Reply:
thanks - resolved.